pkg/resources/network_policy_attachment_acceptance_test.go

```diff
@@ -49,6 +50,20 @@ func TestAcc_NetworkPolicyAttachment(t *testing.T) {
 			},
 		},
 	})
+	resource.ParallelTest(t, resource.TestCase{
+		Providers:    acc.TestAccProviders(),
+		PreCheck:     func() { acc.TestAccPreCheck(t) },
+		CheckDestroy: nil,
+		Steps: []resource.TestStep{
+			{
+				Config: networkPolicyAttachmentConfigAccount(policyNameAccount),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("snowflake_network_policy_attachment.test", "network_policy_name", policyNameAccount),
+					resource.TestCheckResourceAttr("snowflake_network_policy_attachment.test", "set_for_account", "true"),
```

Comment:
nit-picking (just a comment, not required change):
this test initially fails on destroy (before the fix) but this is implicit, the reader is not aware of the use case being tested. It would be more readable to either:
- add CheckDestroy explicitly requiring network policy attachment to be unset or...
- add a second test step with just a network policy and no snowflake_network_policy_attachment

That way our intent is clear.

Reply:
I created a CheckDestroy function but sorry I am not sure I understand what the problem was. Probably it's just that it was a while ago and I don't remember the context. 
Is it OK now?